src/storage/documentService/documentService.js

```diff
@@ -0,0 +1,59 @@
+const verify = require("@govtechsg/oa-verify");
+const uuid = require("uuid/v4");
+const { encryptString } = require("./crypto");
+const config = require("../config");
+const { put, get, remove } = require("../dynamoDb");
+
+const DEFAULT_TTL = 1000 * 60 * 5; // 5 Minutes
```

Comment:
Thought we decided to use like 1 hour or something? (I know i wrote this)

Reply:
changed to 1hr